aws/table_aws_transfer_server.go

```diff
@@ -0,0 +1,260 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/transfer"
+	"github.com/aws/aws-sdk-go-v2/service/transfer/types"
+
+	transferv1 "github.com/aws/aws-sdk-go/service/transfer"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsTransferServer(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_transfer_server",
+		Description: "AWS Transfer Server",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("server_id"),
+			Hydrate:    getTransferServer,
```

Comment:
Please handle the not found error code([Add ignore config](https://github.com/turbot/steampipe-plugin-aws/blob/main/aws/table_aws_acm_certificate.go#L30)) for the API call. The `DescribeService` API throws [ResourceNotFoundException](https://docs.aws.amazon.com/transfer/latest/userguide/API_DescribeServer.html) error when a resource is not found by the AWSTransfer Family service.